docs/src/main/asciidoc/qute.adoc

```diff
@@ -171,6 +171,37 @@ public class ItemResource {
 <2> Make the `Item` object accessible in the template.
 <3> A static template extension method can be used to add "computed properties" to a data class. The class of the first parameter is used to match the base object and the method name is used to match the property name.
 
+== Built-in Template Extension methods
```

Comment:
I think that we should move this section to the reference guide instead: https://github.com/quarkusio/quarkus/blob/master/docs/src/main/asciidoc/qute-reference.adoc

Reply:
The Built in template methods work only if you use the `qute` extension in Quarkus, they are not available if you use Qute standalone, that's why I kept it in `qute.adoc`, but I am fine to move it if you think it's better

Reply:
`qute.adoc` is a simple quickstart guide, `qute-reference.adoc` is a reference guide that contains both the core/standalone and quarkus-specific features.